socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -46,41 +40,66 @@
 <script>
     import Vue from "vue"
 
+
     export default Vue.component("author-bar", {
-        props: {
-            handle: {type: String, required: true},
-            name: {type: String, required: true},
-            guid: {type: String, required: true},
+        props:    {
+            handle:                   {type: String, required: true},
+            name:                     {type: String, required: true},
+            guid:                     {type: String, required: true},
             currentBrowsingProfileId: {type: String, required: true},
-            homeUrl: {type: String, required: true},
-            absoluteUrl: {type: String, required: true},
-            imageUrlSmall: {type: String, required: true},
-            isUserAuthor: {type: Boolean, required: true},
-            isUserLocal: {type: Boolean, required: true},
-            isUserFollowingAuthor: {type: Boolean, required: true},
-            isUserAuthentificated: {type: Boolean, required: true}
+            homeUrl:                  {type: String, required: true},
+            absoluteUrl:              {type: String, required: true},
+            imageUrlSmall:            {type: String, required: true},
+            isUserAuthor:             {type: Boolean, required: true},
+            isUserLocal:              {type: Boolean, required: true},
+            isUserFollowingAuthor:    {type: Boolean, required: true},
+            isUserAuthentificated:    {type: Boolean, required: true},
         },
         data() {
-            return {showProfileBox: false}
+            return {
+                showProfileBox:         false,
+                localIsUserFollowingAuthor: this.isUserFollowingAuthor,
```

Comment:
Vue doesn't distinguish state and props? That is a bit sad :(

How about `_isUserFollowingAuthor` so prefixing local state with `_`? Prefixing with `local` sounds a bit dirty, since a user can be "local" etc. The word has strong other meanings. Might end up with a variable `localLocal...` somewhere.